src/sklearn_evaluation/plot/classification.py

```diff
@@ -57,7 +60,7 @@ def __init__(self, a, b, target_names) -> None:
 
 
 class ConfusionMatrix(Plot):
-    @SKLearnEvaluationLogger.log(feature="plot", action="confusion-matrix-init")
+    @tel_cm.log_call(log_args=True, ignore_args={"y_true", "y_pred", "cm"})
```

Comment:
I hope these are stored in the metadata object, else it'll make it hard to join both event types